site2/website/data/users.js

```diff
@@ -0,0 +1,127 @@
+module.exports = [
+    {
+        name: 'Narvar',
+        url: 'https://corp.narvar.com/',
+        logo: 'https://corp.narvar.com/wp-content/themes/narvarinc/dist/images/narvar-logo.svg',
+    },
+    {
+        name: 'Giggso',
+        url: 'https://giggso.com/',
+        logo: 'https://pbs.twimg.com/profile_banners/4179602009/1470596851/600x200'
+    },
+    {
+        name: 'STICorp',
+        url: 'https://www.sticorp.com.br/',
+        logo: 'https://www.sticorp.com.br/wp-content/uploads/2015/09/logo-horizontal.png',
+    },
+    {
+        name: 'Verizon Media',
+        url: 'https://www.verizonmedia.com/',
+        logo: 'https://upload.wikimedia.org/wikipedia/commons/thumb/0/09/Verizon_Media_2019_logo.svg/1200px-Verizon_Media_2019_logo.svg.png'
+    },
+    {
+        name: 'Yahoo! Japan',
```

Comment:
Could you modify to Yahoo! **JAPAN** ?
(It's YJ's formal service name)

Reply:
Sure, fixed

Reply:
Thanks!